browser/extensions/BUILD.gn

```diff
@@ -41,10 +41,13 @@ source_set("extensions") {
   deps = [
     "//brave/browser/resources:brave_extension_grit",
     "//brave/common/extensions/api",
-    "//brave/components/brave_rewards/resources/extension:resources",
+    "//brave/components/brave_rewards/resources/extension:extension_generated_resources",
+    "//brave/components/brave_rewards/resources/extension:static_resources",
     "//brave/components/brave_sync",
-    "//brave/components/brave_sync:resources",
-    "//brave/components/brave_webtorrent:resources",
+    "//brave/components/brave_sync:generated_resources",
```

Comment:
can you add a `group("resources")` target for these with public_deps on static/generated?

Reply:
just makes it a little easier for other targets that depend on them

Reply:
we can also repak the static/generated into a single pak so we only have to reference one here

Reply:
can be a follow-up